firmware/config/boards/microrusefi/board_configuration.cpp

```diff
@@ -208,16 +212,14 @@ void setBoardDefaultConfiguration() {
 	setupDefaultSensorInputs();
 
 
-	// SPI for SD card
```

Comment:
Shall we keep spi2=true and improve the comment to "SPI for official 0.6.0 SD card"?

since that's now part of official HW i think we want spi2 enabled by default

Reply:
it's enabled in the override